helm/nats-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "nats.fullname" . }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: "{{ template "nats.name" . }}"
+      chart: "{{ template "nats.chart" . }}"
+      release: "{{ .Release.Name }}"
+  template:
+    metadata:
+      labels:
+        app: "{{ template "nats.name" . }}"
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+      {{- if .Values.podLabels }}
+        {{ .Values.podLabels}}
+      {{- end }}
+      {{- if .Values.podAnnotations }}
+      annotations:
+        {{ .Values.podAnnotations }}
+      {{- end }}
+    spec:
+      updateStrategy:
+        type: {{ .Values.updateStrategy }}
+        {{- if .Values.rollingUpdatePartition }}
+        rollingUpdate:
+          partition: {{ .Values.rollingUpdatePartition }}
+        {{- end }}
+      {{- if .Values.rbacEnabled }}
+      serviceAccountName: nats-operator
+      {{- end }}
+      containers:
+      - name: nats-operator
+        image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}      
+        imagePullPolicy: {{ .Values.pullPolicy  }}
+        env:
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: monitoring
```

Comment:
Not sure that the NATS Operator has a monitoring port for liveness/readiness probe (may have missed it... if so we should include in the examples...)

Reply:
Ok so we can use something for the readiness probe, adding to the examples: https://github.com/nats-io/nats-operator/pull/63

Reply:
I pushed a new commit changing the port to `8080`. I tested it in my side and I can poke `:8080/readyz` succesfully